samples/Extensions/CosmosDB/CosmosDBFunction.cs

```diff
@@ -33,6 +34,7 @@ public static class CosmosDBFunction
 
             return null;
         }
+        //</docsnippet_exponential_backoff_retry_example>
```

Comment:
24 lines of code. Curious to know whether it would create vertical scrollbar in the docs page for the snippet?